### DISCUSSION TITLE
Is level for money placed on /is bank.

### DISCUSSION THREAD

**Discussion by SwartZCoding** [BG-Software-LLC/SuperiorSkyblock2#977] [Pending Feature Requests]
All on my title :)

Add feature to permit to won points when players placed money on /is bank.

If this money is removed, points decrease.
If money is added, points increase.

**Comment by OmerBenGera** [comment]
Money goes towards the worth of the island.

**Reply by SwartZCoding** [reply]
Oh shit, i see that. It's on module and not main config file. Sorry ^^.

**Comment by SwartZCoding** [comment]
Update : ok money goes towards the worth of the island but she doesn't permit to up island level ??

Because when i deposit money on my island bank i won value. But level stay the same 😅

If normally yes, i think i found an issue ^^

**Reply by OmerBenGera** [reply]
> Update : ok money goes towards the worth of the island but she doesn't permit to up island level ??
> 
> Because when i deposit money on my island bank i won value. But level stay the same 😅
> 
> If normally yes, i think i found an issue ^^

This is intended. The worth system is to determine the worth of the island, and its obvious that the bank will go towards this count. The level system is there to give another type of evaluation of the island, that is not related to worth value of blocks, but to another type of value.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
